libcxxabi/test/native/x86_64/lpstart-zero.pass.sh.s

```diff
@@ -2,7 +2,7 @@
 # RUN: %t.exe
 
 # REQUIRES: linux && target={{x86_64-.+}}
-# UNSUPPORTED: no-exceptions
+# UNSUPPORTED: no-exceptions || android
```

Comment:
@rprichard  What would be the right target-based annotation here?

```
// UNSUPPORTED: target={{.+}}-android{{.+}}
```

?

Reply:
I think I would change the suffix to use `{{.*}}` to also match a triple without the version number suffix, just for completeness:

```
// UNSUPPORTED: target={{.+}}-android{{.*}}
```

The two unversioned Android annotations I currently see in libc++ also use a `{{.*}}` suffix:

```
// XFAIL: LIBCXX-ANDROID-FIXME && target={{i686|x86_64}}-{{.+}}-android{{.*}}
```
```
// UNSUPPORTED: target=i686-{{.+}}-android{{.*}}
```